tests/sles4sap/saptune.pm

```diff
@@ -26,18 +24,7 @@ sub tuned_is {
 sub run {
     my ($self) = @_;
 
-    # Test has to work differently on x86_64 and ppc64le. Will verify test
-    # is running on ppc64le via the OFW variable
-    my $is_ppc64le = get_var('OFW');
```

Comment:
So test failed in ppc64le (https://openqa.suse.de/tests/3226218#step/saptune/12) as `saptune` is not installed by default in 12-SP5 and gnome test runs without a previous run of `sles4sap/patterns`. I think we could add a `zypper in -y saptune` for that scenario (ppc64le and sles4sap < 15), but I'm not completely certain we're introducing a false positive somewhere with this.

Reply:
Agreed.